scripts/e2e-stop.sh

```diff
@@ -0,0 +1,31 @@
+MY_PATH="`dirname \"$0\"`"              # relative
+MY_PATH="`( cd \"$MY_PATH\" && pwd )`"  # absolutized and normalized
+if [ -z "$MY_PATH" ] ; then
+  # error; for some reason, the path is not accessible
+  # to the script (e.g. permissions re-evaled after suid)
+  exit 1  # fail
+fi
+
+
+export PORTAL_SOURCE_DIR=$PWD;
```

Comment:
how come we do all this in stop script?

Reply:
`stop.sh` calls `cd $E2E_WORKSPACE/cbioportal-docker-compose` before executing the docker commands